.github/workflows/build-deploy-ec2.yml

```diff
@@ -0,0 +1,65 @@
+name: Build and Deploy to EC2
+on: workflow_dispatch
+jobs:
+  build-deploy:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.10"]
+        node-version: [16.x]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install Python dependencies
+        working-directory: ./api
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install flake8 tox
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+          if [ -f test-requirements.txt ]; then pip install -r test-requirements.txt; fi
```

Comment:
Aren't `requirements.txt` and `test-requirements.txt` a requirement to properly build and test the application? It might be better to fail if these files aren't found. I'd suggest removing the `-f` check and just do:

```
    pip install -r requirements.txt
    pip install -r test-requirements.txt
```

Reply:
That's better! Thank you.